### DISCUSSION TITLE
Handling db_default?

### DISCUSSION THREAD

**Discussion by shangxiao** [encode/django-rest-framework#9257] [Potential Issue]
Django 5.0 introduces `db_default`, this now raises the question: should DRF be aware of fields with db_default set?

Aside from whether DRF could potentially evaluate a `db_default` expression, it does have an affect on the `required` attribute during introspection, [since required is determined based on the presence of field default](https://github.com/encode/django-rest-framework/blob/df89f32b88022828ca5d1b5e04be36d54f506e6c/rest_framework/fields.py#L310).

The workaround for now is to manually set `required` to `False` in `extra_kwargs` if you have a `db_default`.

**Comment by auvipy** [comment]
may be we could see/try with some example use cases of the new features and then decide based on the outcomes?
